_layouts/product-index.html.slim

```diff
@@ -1,61 +1,60 @@
 ---
```

Comment:
With this file it looks like everything has been indented by two spaces. Can we clear this so we can see more clearly what the changes were?